18 - Apple HomePage/css/style.css

```diff
@@ -0,0 +1,382 @@
+*{
+    font-family: system-ui, -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    outline: none;
```

Comment:
move this file directly in the "apple homepage' folder and delete the folder "CSS".